pkg/cmd/backuprestore/backupserver.go

```diff
@@ -0,0 +1,105 @@
+package backuprestore
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"k8s.io/klog/v2"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var shutdownSignals = []os.Signal{os.Interrupt, syscall.SIGTERM}
+
+type backupServer struct {
+	schedule string
+	timeZone string
+	enabled  bool
+	//scheduler *tasker.Tasker
```

Comment:
please remove the commented out code in this file

Reply:
done 👍🏽